Editor/Bootstrap/BootstrapSettings.cs

```diff
@@ -24,6 +24,7 @@ internal class BootstrapSettings
         public const int SelectionMode = 1;
         public const int NoneLifeCyclePolicyIndex = 0;
         private const int DefaultLifeCyclePolicyIndex = 0;
+        private const string CannotCreateErrorMessage = "Cannot Create Bucket, Bucket Name is Empty";
```

Comment:
minor: rename variable name to `BucketNameEmptyErrorMessage` so the name is more specific